cinema/views.py

```diff
@@ -57,20 +54,18 @@
 ):
     queryset = CinemaHall.objects.all()
     serializer_class = CinemaHallSerializer
-    authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
+
 
 
 class MovieViewSet(
     mixins.ListModelMixin,
     mixins.CreateModelMixin,
     mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
```

Comment:
The `MovieViewSet` includes the `UpdateModelMixin`, which allows for updating movie instances. Ensure that the update functionality is tested and that appropriate permissions are enforced.